arbitrum-docs/how-arbitrum-works/07-interactive-fraud-proofs.mdx

```diff
@@ -0,0 +1,499 @@
+
+This does mean that transactions that go through the delayed inbox will take longer to get finality. Their time to finality will roughly double, because they will have to wait one finality period for promotion, then another finality period for the Ethereum transaction that promoted them to achieve finality.
+
+This is the basic tradeoff of having a Sequencer: if your message uses the Sequencer, finality is C blocks faster; but if your message doesn't use the Sequencer, finality is C blocks slower. This is usually a good tradeoff, because most transactions will use the Sequencer; and because the practical difference between instant and 10-minute finality is bigger than the difference between 10-minute and 20-minute finality.
+
+So a Sequencer is generally a win, if the Sequencer is well behaved.
+
+## If the Sequencer is malicious...
+
+A malicious Sequencer, on the other hand, could cause some pain. If it refuses to handle your transactions, you're forced to go through the delayed inbox, with longer delay. And a malicious Sequencer has great power to front-run everyone's transactions, so it could profit greatly at users' expense.
+
+On Arbitrum One, Offchain Labs [currently](https://docs.arbitrum.foundation/state-of-progressive-decentralization) runs a Sequencer which is well-behaved--we promise!. This will be useful but it's not decentralized. Over time, we'll switch to decentralized, fair sequencing, as described below.
+
+Because the Sequencer will be run by a trusted party at first, and will be decentralized later, we haven't built in a mechanism to directly punish a misbehaving Sequencer. We're asking users to trust the centralized Sequencer at first, until we switch to decentralized fair sequencing later.
+
+## Decentralized fair sequencing
+
+Viewed from 30,000 feet, decentralized fair sequencing isn't too complicated. Instead of being a single centralized server, the Sequencer is a committee of servers, and as long as a large enough supermajority of the committee is honest, the Sequencer will establish a fair ordering over transactions.
+
+How to achieve this is more complicated. Research by a team at Cornell Tech, including Offchain Labs CEO and Co-founder Steven Goldfeder, developed the first-ever decentralized fair sequencing algorithm. With some improvements that are under development, these concepts will form the basis for our longer-term solution, of a fair decentralized Sequencer.
+
+<!-- -->
+
+<!-- from fraud-proofs/challenge-manager -->
+
+<!--import ChallengeManagerDiagram from '../../diagrams/_challenge-manager.mdx'; -->
+
+## ChallengeManager
+
+The `ChallengeManager` arbitrates challenge games. Here's a diagram of the challenge state machine:
+
+<!--         <ChallengeManagerDiagram />        -->
```

Comment:
Missing diagram

Reply:
Vercel was giving me issues with this specifically, I had to comment it out to get it to build. Will check with Gael 👍